app/views/shared/_table_filter.html.erb

```diff
@@ -1,7 +1,10 @@
-<div class="toolbar">
+<div id="tablefilter-<%= SecureRandom.hex %>" class="tablefilter form-group pull-right">

```

Comment:
Remove the tag id, I guess you introduced it for the map of debounced functions, which is not needed.